ztp/policygenerator/policyGen/testData/GenericSourceFiles/GenericCRWithTrailingSeparators.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Perhaps having some leading empty docs would also be a good idea for a test?

Reply:
Good idea! I have added that.